### ISSUE TITLE
[DL3025] False positive when there is a space before a comma

### ISSUE THREAD

**Issue by charlyx** [hadolint/hadolint#307]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

It should PASS.

### Actual behavior

It FAILS.

### Steps to reproduce the behavior

Hi 👋

First of all thanks for your work!
I use hadolint everywhere I use Docker 😄 

I encountered a bug though: hadolint indicates a DL3025 error when there is a space before a comma.
There might be other false positive I haven't tested all possibilities.

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.16.0-0-g475ad73
```

**Dockerfile** (if relevant)

```Dockerfile
# Fails
CMD ["node", "-r", "esm" ,"src/index.js"]
# Pass
CMD ["node", "-r", "esm", "src/index.js"]
```

**Additional environment details (OS, stack version, etc.)**

**Comment by lorenzo** [comment]
Thanks for your report, I'll look into this
